packages/jupyter-ai/package.json

```diff
@@ -78,19 +78,15 @@
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-markdown": "^8.0.6",
-    "react-syntax-highlighter": "^15.5.0",
-    "rehype-katex": "^6.0.2",
-    "remark-math": "^5.1.1"
+    "react-dom": "^18.2.0"
```

Comment:
You need to declare `"@jupyterlab/rendermime": "^4"` as a direct dependency. This isn't strictly necessary since `jupyter labextension build` will pull whatever copy exists in your Python environment, but this `package.json` does get read by Webpack and can cause weird build errors randomly if the dependency list is incomplete. Sadly I lack data to back this up, just speaking from experience.

Reply:
Didn't know about that. Included in `dependencies`.